### PR TITLE
build: use remote actions for TS/NG compilations on CircleCI

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

File: .circleci/bazel.rc [hunk 1]
```diff
@@ -28,3 +28,16 @@ test --flaky_test_attempts=2
...
+
+# We have seen some flakiness in using TS workers on CircleCI
+# https://angular-team.slack.com/archives/C07DT5M6V/p1562693245183400
+# > failures like `ERROR: /home/circleci/ng/packages/core/test/BUILD.bazel:5:1:
+# > Compiling TypeScript (devmode) //packages/core/test:test_lib failed: Worker process did not return a WorkResponse:`
+# > I saw that issue a couple times today.
+# > Example job: https://circleci.com/gh/angular/angular/385517
+# We expect that TypeScript compilations will parallelize wider than the number of local cores anyway
+# so we should saturate remote workers with TS compilations
+# TODO(jperott): if/when we enable remote cache on CI, we would expect
```
Comment (josephperrott) [inline .circleci/bazel.rc]: Remote cache is already [enabled](https://github.com/angular/angular/blob/master/.circleci/config.yml#L63) on CI

Though, I am not sure what the interop between RBE/caching and keeping a daemon up is.  If everything is sent out to RBE does the RBE environment holds its state to allow for the daemon to be used?  Or is it a fresh environment everytime anyway?

**Review by kara — APPROVED** [review]
LGTM

**Comment by kara** [comment]
merge-assistance: global

**Comment by alexeagle** [comment]
Thanks for pushing it through @josephperrott

**Pull request closed without merging**